src/main/java/ledger/controller/DbController.java

```diff
@@ -31,6 +31,7 @@
     private List<CallableMethodVoidNoArgs> transactionSuccessEvent;
     private List<CallableMethodVoidNoArgs> accountSuccessEvent;
     private List<CallableMethodVoidNoArgs> payeeSuccessEvent;
+    private List<CallableMethodVoidNoArgs> ignoredExpSuccessEvent;
```

Comment:
Do we need this? Can't find anywhere its used and I'm not sure I see a use case.